zellij-server/src/wasm_vm.rs

```diff
@@ -212,6 +212,8 @@ fn start_plugin(
         .unwrap()
         .map_dir("/data", plugin_own_data_dir.as_path())
         .unwrap()
+        .map_dir("/tmp", ZELLIJ_TMP_DIR.as_path())
```

Comment:
Maybe it would be good to support STATE/CACHE dirs in the future?
For now I would be OK with this.
But probably something we want to restrict again at some point?
cc @TheLostLambda